.github/stale.yml

```diff
@@ -17,6 +17,9 @@ exemptLabels:
   - optimization
   - "needs roadmapped"
```

Comment:
Why does this one have ""s and the other one doesn't?

Reply:
These are based on what our labels are in Github, so not sure why one has an s and one doesn't!